src/interface/stmt.ml

```diff
@@ -195,6 +195,13 @@ module type Logic = sig
       i.e f is a function symbol with arguments [args], and which returns the value
       [body] which is of type [ret]. *)
 
+  val pred_def    : ?loc:location -> id -> term list -> term list -> term -> term -> t
+  (** Symbol definition. [pred_def p vars args ret body] means that "p(args) = (body : bool)",
```

Comment:
does that mean that `ret` is unused (since it's usually the return type of the symbol).

Reply:
Yes